### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by ETStareak**
Hello！
It's such a pity that I didn't know the new version at the first time.As @Joccob  (the people who translated 0.4 ) decided not keep on translating,he asked me to work on this.(He also asked me to translate GalactiCraft4)
@DYColdWind  won't keep on translating MatterOverdrive,and I will keep on translating.

### CODE REVIEW

File: src/main/resources/assets/matteroverdrive/lang/zh_cn.lang [hunk 1]
```diff
@@ -720,12 +720,12 @@ quest.one_true_love.title=真挚的爱
 quest.one_true_love.info=我真的好开心！你甚至无法想象这一刻我是多么快乐！Merry终于同意嫁给我，你也许不知道我为这一刻付出了多少心血。不过这一切归根到底还是得力于她的父亲给予我们的祝福，我知道以我现在的能力足以给她一个美好生活，但我还是想给她一些特别的礼物。我希望你能帮助我找到一颗钻石，这样我就可以将钻石镶饰在戒指以便给她戴上，我相信她一定会记住这难忘的一天。另外，作为回报，我很乐意尽我最大的能力帮助你。
 quest.one_true_love.objective.0=挖取 $mineAmount/$maxMineAmount 钻石
 quest.weapons_of_war.title=起义的开始
-quest.weapons_of_war.info=当初那个任人宰割的日子将一去不复返了！我们向往自由和平的人类将与机械人展开一场孤身一注的战争，是时候该结束机械人接管世界的局面了。只要我们意志坚定，就没有什么是不可战胜的。不过，在战争开始之前。我们可能需要先酝酿一下准备好一些趁手的武器和保护自己的盔甲。
+quest.weapons_of_war.info=当初那个任人宰割的日子将一去不复返了！我们向往自由和平的人类将与机械人展开一场孤身一注的战争，是时候该结束博格们控制世界的局面了。只要我们意志坚定，就没有什么是不可战胜的。不过，在战争开始之前。我们可能需要先酝酿一下准备好一些趁手的武器和保护自己的盔甲。
```
Comment (DYColdWind) [inline src/main/resources/assets/matteroverdrive/lang/zh_cn.lang]: ‘是时候该结束博格们控制世界的局面了’
博格（Borg）？

Comment (ETStareak) [reply src/main/resources/assets/matteroverdrive/lang/zh_cn.lang]: 忘改回去了。

Comment (ETStareak) [reply src/main/resources/assets/matteroverdrive/lang/zh_cn.lang]: 忘改回去了。

Comment (ETStareak) [reply src/main/resources/assets/matteroverdrive/lang/zh_cn.lang]: Finished.

File: src/main/resources/assets/matteroverdrive/info/zh_CN/item.ion_sniper.xml [hunk 2]
```diff
@@ -26,14 +26,14 @@
         <text class="matter-text text-block">由于$itemName的特性允许它在很远的地方射击敌人，因此它也是远程武器的最大受益者。尤其是安装了[item name=sniper_scope]。</text>
         <text class="matter-text text-block">残暴机械人可以使用这种远程能量武器，这将是非常致命的。并且它们可以远距离狙击。</text>
         <text class="matter-text text-block">像大多数能量武器一样，$itemName能够通过[item name=energy_pack]来快速充能，
-            如果想要持续不停的射击，则可以在[block name=weapon_station]上使用武器模块来进行升级。</text>
+            如果想要连续的射击，则需在[block name=weapon_station]上使用武器模块来进行升级。</text>
         <text class="matter-text text-block">$itemName可以使用所有升级包括狙击瞄准镜。</text>
```
Comment (DYColdWind) [inline src/main/resources/assets/matteroverdrive/info/zh_CN/item.ion_sniper.xml]: -item.matteroverdrive.sniper_scope.name=狙击瞄准镜
+item.matteroverdrive.sniper_scope.name=狙击镜

File: src/main/resources/assets/matteroverdrive/info/zh_CN/item.phaser_rifle.xml [hunk 3]
```diff
@@ -25,13 +25,13 @@
     <page class="padded-page">
         <title size="2" class="matter-text small-title" style="margin-bottom:6px">Customization</title>
         <text class="matter-text text-block">像大多数能量武器一样，$itemName能够通过[item name=energy_pack]来快速充能，
-            如果想要持续不停的射击，则可以在[block name=weapon_station]上使用武器模块来进行升级。</text>
+            如果想要连续射击，则需在[block name=weapon_station]上使用武器模块来进行升级。</text>
         <text class="matter-text text-block">$itemName可以使用所有升级包括狙击瞄准镜。</text>
```
Comment (DYColdWind) [inline src/main/resources/assets/matteroverdrive/info/zh_CN/item.phaser_rifle.xml]: -item.matteroverdrive.sniper_scope.name=狙击瞄准镜
+item.matteroverdrive.sniper_scope.name=狙击镜

File: src/main/resources/assets/matteroverdrive/info/zh_CN/weapon.modules.colors.xml [hunk 4]
```diff
@@ -19,8 +19,8 @@
 <entry stylesheet="matteroverdrive:info/styles/guide.css">
     <page class="padded-page">
         <title size="2" class="matter-text small-title">Weapon Color Modules</title>
```
Comment (DYColdWind) [inline src/main/resources/assets/matteroverdrive/info/zh_CN/weapon.modules.colors.xml]: 看来我当时忘了翻译了，它应该叫‘武器颜色模块’？

File: src/main/resources/assets/matteroverdrive/lang/zh_cn.lang [hunk 5]
```diff
@@ -156,21 +156,21 @@ tile.network_router.name=物质网络路由器
...
-tile.dilithium_ore.name=双锂矿石
+tile.dilithium_ore.name=二锂矿石
 tile.tritanium_ore.name=三钛合金矿石
 tile.tritanium_block.name=三钛合金块
 tile.heavy_matter_pipe.name=重型物质运输管道
 tile.heavy_matter_pipe.details=运输等离子态物质
-tile.holo_sign.name=全息显示板
-tile.force_glass.name=三钛合金玻璃
+tile.holo_sign.name=全息显示屏
+tile.force_glass.name=三钛玻璃
```
Comment (DYColdWind) [inline src/main/resources/assets/matteroverdrive/lang/zh_cn.lang]: ‘+tile.force_glass.name=三钛玻璃’
在en_us中以修改为‘tile.industrial_glass.name=Industrial Glass‘’

Comment (ETStareak) [reply src/main/resources/assets/matteroverdrive/lang/zh_cn.lang]: emm那就工程玻璃？

File: src/main/resources/assets/matteroverdrive/info/zh_CN/item.ion_sniper.xml (hunk 2 above)
Comment (ETStareak) [reply src/main/resources/assets/matteroverdrive/info/zh_CN/item.ion_sniper.xml]: 哪来的瞄准这词啊，这是个专有词组，狙击这一动作本身包含了瞄准，语义重复了。

File: src/main/resources/assets/matteroverdrive/info/zh_CN/weapon.modules.colors.xml (hunk 4 above)
Comment (ETStareak) [reply src/main/resources/assets/matteroverdrive/info/zh_CN/weapon.modules.colors.xml]: 我觉得武器一词可以省略的。

Comment (ETStareak) [reply src/main/resources/assets/matteroverdrive/info/zh_CN/weapon.modules.colors.xml]: 我觉得武器一词可以省略的。

Comment (DYColdWind) [reply src/main/resources/assets/matteroverdrive/info/zh_CN/weapon.modules.colors.xml]: 恩，可以。

**Review by DYColdWind — APPROVED** [review]

**Review by Muirrum — APPROVED** [review]
I'm going to trust you

**Pull request merged**